src/InstallationValidator.Core/Reporting/BatchComparisonResultTeXBuilder.cs

```diff
@@ -38,20 +40,5 @@ public override void Build(BatchComparisonResult comparisonResult, OSPSuiteTrack
 
          _teXBuilderRepository.Report(objectsToReport, buildTracker);
       }
-
-      private ValidationState validationResultFor(BatchComparisonResult comparisonResult)
-      {
-         return comparisonResult.State;
-      }
-
-      private string secondComparisonFolder(BatchComparisonResult comparisonResult)
-      {
-         return comparisonResult.FolderPath2;
```

Comment:
I am not sure that these methods returning directly a property of an object make the code more readable.